### PR TITLE
Summer data

### PR DESCRIPTION
**Pull request by Slesarew**

### CODE REVIEW

File: data/liquid/2023/2023-06-30-4.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+protocol.name = "propagation"
+protocol.version = "0.2.0"
+parent = "2023-06-28-3"
+time = "2023-06-30"
+batch.size = 16
+wort.value = 150.0
+wort.error = 0.1
+wort.unit = "g"
+
+notes = "one flask (smaller volume) remains at 24+-2C in incubator"
```
Comment (varovainen) [inline data/liquid/2023/2023-06-30-4.toml]:
```suggestion
notes = "one flask (smaller volume) remains at 24+-1C in incubator"
```

File: data/liquid/2023/2023-07-01-3.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+time = "2023-07-01"
+batch.size = 19
+pitch.value = 13
+pitch.error = 0.01
+pitch.unit = "ml"
+wort.value = 50.00
+wort.error = 0.05
+wort.unit = "g"
+
+notes = "first flask is kept at 24+-2C"
```
Comment (varovainen) [inline data/liquid/2023/2023-07-01-3.toml]:
```suggestion
notes = "first flask is kept at 24+-1C"
```

File: data/liquid/2023/2023-07-03-4.toml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+medium = "liquid"
+protocol.name = "propagation"
+protocol.version = "0.2.0"
+parent = "2023-07-01-3"
+time = "2023-07-03"
+wort.value = 150.0
+wort.error = 0.1
+wort.unit = "g"
+
+notes = "first flask has 182.10+-0.01g wort, kept at 24+-2C"
```
Comment (varovainen) [inline data/liquid/2023/2023-07-03-4.toml]:
```suggestion
notes = "first flask has 182.10+-0.01g wort, kept at 24+-1C"
```

**Review by varovainen — CHANGES_REQUESTED** [review]

**Comment by Zhenya-Barannik** [comment]
> Building using data from commit [439357618144849e6a5345d0f88ee55a49cfaaef](https://github.com/Alzymologist/yeast/commit/439357618144849e6a5345d0f88ee55a49cfaaef) Generated image: https://raw.githubusercontent.com/Alzymologist/yeast/genealogy-preview/genealogy-439357618144849e6a5345d0f88ee55a49cfaaef.png Generated Image

This image was rewritten due to testing in other pull-request. Rewriting is fixed in the newer version of the workflow (only at feature-2 branch right now). Names for images will be 
"genealogy-${COMMIT_SHA}-${TARGET_BRANCH}.png"
instead of
"genealogy-${COMMIT_SHA}.png"

**Review by varovainen — APPROVED** [review]

**Pull request merged**